src/main/msc/emfat_file.c

```diff
@@ -411,40 +427,46 @@ void emfat_init_files(void)
     }
 #endif
 
+    flashInit(flashConfig());
```

Comment:
They're moved up into `init.c` which is now the only place `emfat_init_files()` is called.
```
#if defined(USE_FLASHFS) && defined(USE_FLASH_CHIP)
        // If the blackbox device is onboard flash, then initialize and scan
        // it to identify the log files *before* starting the USB device to
        // prevent timeouts of the mass storage device.
        if (blackboxConfig()->device == BLACKBOX_DEVICE_FLASH) {
            emfat_init_files();
        }
#endif
```

Reply:
The blinking duration will depend on how much flash is actually used. But even if full 8MB will probably only be a second or two.

Reply:
While looking at this I found another problem: #9253

This could be rebased on that to simplify it a little. Really all that's needed is to test `USE_FLASHFS` (but needs the other fix).

Reply:
I thought this was more complex than what it is now...

Reply:
Previously this was in the USB device initialization callback from the STM library:
```
#ifdef USE_FLASHFS 
#ifdef USE_FLASH_CHIP
    flashInit(flashConfig());
#endif
    flashfsInit();
#endif
    emfat_init_files();
```
Effectively this is moved up into `init()` to run before the USB device is started instead of after.